app/message-mark-as-unread/server/unreadMessages.js

```diff
@@ -2,18 +2,19 @@ import { Meteor } from 'meteor/meteor';
 
 import logger from './logger';
 import { Messages, Subscriptions } from '../../models';
+import { Message } from '../../../server/sdk';
 
 Meteor.methods({
-	unreadMessages(firstUnreadMessage, room) {
+	unreadMessages(firstUnreadMessage, rid) {
```

Comment:
No need to change this file, it's not returning a list os messages. It's only marking the room as read.